pipelines/rh-push-to-external-registry/README.md

```diff
@@ -17,6 +17,10 @@ Tekton pipeline to release Red Hat Snapshots to an external registry. This pipel
 | taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/redhat-appstudio/release-service-catalog.git |
 | taskGitRevision | The revision in the taskGitUrl repo to be used | No | - |
 
+### Changes in 2.2.0
+- modified the pipeline to dynamically source the `data.json`, `snapshot_spec.json` and
```

Comment:
Capital M to match the other entries please

Reply:
Fixed, thanks!